core/src/main/java/gatt/Debug.kt

```diff
@@ -101,6 +93,14 @@ internal fun GattConnectionStatus.asGattConnectionStatusString() = when (this) {
     else -> "GATT_CONN_UNKNOWN"
 }.let { name -> "$name($this)" }
 
+internal fun GattConnectionState.asGattConnectionStateString() = when (this) {
+    STATE_DISCONNECTING -> "STATE_DISCONNECTING"
+    STATE_DISCONNECTED -> "STATE_DISCONNECTED"
+    STATE_CONNECTING -> "STATE_CONNECTING"
+    STATE_CONNECTED -> "STATE_CONNECTED"
+    else -> "STATE_UNKNOWN"
+}.let { name -> "$name($this)" }
+
```

Comment:
No code change, just moved to match ordering of **status** and **state** within the rest of the code base.